packages/cli/README.md

```diff
@@ -3107,7 +3107,7 @@ There are a lot of details left out - check out the full example app for a worki
 
 ### Why doesn't Zapier support newer versions of Node.js?
 
-We run your code on AWS Lambda, which only supports a few [versions](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html) of Node (the latest of which is `v14`. As that updates, so too will we.
+We run your code on AWS Lambda, which only supports a few [versions](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html) of Node (the latest of which is `v16`. As that updates, so too will we.
```

Comment:
Same comment as above, we are not currently in parity with Lambda Node releases. Considering the question this is answering, it's probably worth re-wording this...

```suggestion
We run your code on AWS Lambda, which only supports a few [versions](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html) of Node. We periodically update the runtime version of Node to keep up with the latest supported version in AWS Lambda.
```